crates/holochain/tests/tests/graft_records_onto_source_chain.rs

```diff
@@ -180,47 +187,40 @@ async fn grafting() {
     action.action_seq = 2;
     action.prev_action = chain[1].0.clone();
     action.timestamp = Timestamp::from_micros(0);
-    let record = make_record(&conductor.keystore(), action.clone().into()).await;
+    let record = make_record(&conductor_1.keystore(), action.clone().into()).await;
 
     // Insert an invalid action with validation on.
-    let result = conductor
+    let result = conductor_1
         .clone()
-        .graft_records_onto_source_chain(alice.cell_id().clone(), true, vec![record.clone()])
+        .graft_records_onto_source_chain(alice_cell_1.cell_id().clone(), true, vec![record.clone()])
         .await;
 
     // Fork is detected
     assert!(result.is_err());
 
     // Restore and validate the original records
     // (there has been no change at this point, but it helps for clarity to reset the chain anyway)
-    conductor
+    conductor_1
         .clone()
-        .graft_records_onto_source_chain(alice.cell_id().clone(), true, original_records.clone())
+        .graft_records_onto_source_chain(
+            alice_cell_1.cell_id().clone(),
+            true,
+            original_records.clone(),
+        )
         // Restoring the original records is ok because they
         // will pass validation.
         .await
         .expect("Should restore original chain");
 
-    drop(conductor);
+    drop(conductor_1);
 
     // Start a second conductor.
-    let conductor =
+    let mut conductor_2 =
         SweetConductor::create_with_defaults(config, Some(keystore), None::<DynSweetRendezvous>)
             .await;
 
```

Comment:
Same as in the `multi_conductor_chc_sync()` test at the top of this PR, we replace the usage of `install_app_bundle()` here and instead use the new `install_app()` method on the SweetConductor (see below).